v1/formats/BDIV.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<debate-format schema-version="2.2">
+  <name>Bangladeshi Asian Parlimentary</name>
+  <version>1.00</version>
+  <info>
+    <region>Bangladesh</region>
+    <level>University</level>
+    <used-at>National IV</used-at>
```

Comment:
```suggestion
    <used-at>Bangladesh National Intervarsity</used-at>
```
(prefer no abbreviations and specify country if it's part of the tournament name, please modify if the official tournament name is different)